### PR TITLE
Fixes #6567: Add a script to manage node keys on server

### PR DESCRIPTION
**Pull request by peckpeck**
http://www.rudder-project.org/redmine/issues/6567

### CODE REVIEW

**Comment by peckpeck** [comment]
PR updated

**Comment by amousset** [comment]
Should this be added to rudder cli ? (like rudder server debug).

File: rudder-webapp/SOURCES/rudder-keys [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+# Commands
+LDAP_PARAMETERS="-H ldap://${LDAP_SERVER}:${LDAP_PORT}/ -D ${LDAP_USER} -w ${LDAP_PASSWORD} -x"
+
+
+if [ "${ACTION}" = "reset-status" ]
+then
+
+  ldapmodify ${LDAP_PARAMETERS} <<EOF
+dn: nodeId=nodeId=${UUID},${ACCEPTED_NODES_BASE}
```
Comment (amousset) [inline rudder-webapp/SOURCES/rudder-keys]: There are two "nodeId=" in the dn.

Comment (peckpeck) [inline rudder-webapp/SOURCES/rudder-keys]: Oops I thought I had corrected it

**Comment by peckpeck** [comment]
I don't think this should be added to rudder cli.
This is a temporary script for a feature that should be added to the interface and to the API. It will then be time to remove the script.

**Comment by VinceMacBuche** [comment]
After discussing with @fanf  and @peckpeck, we agreed on the fact that those functions should not be in rudder-cli, and that they would be integrated via api functions (and rudder-api-cli) 

However we will upgrade the error messages in ldap-inventory to talk about that script

**Comment by VinceMacBuche** [comment]
Merging!

**Comment by VinceMacBuche** [comment]
Not merging ;) the dn 'nodeid=nodeid=' is not fixed @peckpeck  ;)

**Comment by peckpeck** [comment]
PR updated

**Comment by VinceMacBuche** [comment]
Thanks @peckpeck ! merging

**Pull request merged**